src/reducers/groups.js

```diff
@@ -0,0 +1,9 @@
+export const groupsReceived = (state, action) => ({ ...state, groups: action.payload, groupMap: createGroupMap(action.payload) })
+
+export const createGroupMap = (groupList) => {
+  const groupMap = {}
+  groupList.forEach((group) => groupMap[group.id] = group.label)
+  return groupMap
+}
+
+export const noop = () => {}
```

Comment:
so this exact same code is in `src/reducers` AND in `src/actions` ?

Reply:
do all files need that license comment header?

Reply:
all of the plumbing is needed between the reducers and the actions (still getting it straight myself) ...

and indeed the license is not consistent applied across the project, i'm sure I missed some in my new files